packages/application-generic/src/decorators/user-session.decorator.ts

```diff
@@ -11,28 +14,12 @@ export const UserSession = createParamDecorator((data, ctx) => {
   }
 
   if (req.user) {
-    /**
-     * This helps with sentry and other tools that need to know who the user is based on `id` property.
-     */
-    req.user.id = req.user._id;
-    req.user.username = (req.user.firstName || '').trim();
-    req.user.domain = req.user.email?.split('@')[1];
-
     return req.user;
   }
 
-  if (req.headers) {
-    if (req.headers.authorization) {
-      const tokenParts = req.headers.authorization.split(' ');
-      if (tokenParts[0] !== 'Bearer')
-        throw new UnauthorizedException('bad_token');
-      if (!tokenParts[1]) throw new UnauthorizedException('bad_token');
-
-      const user = jwt.decode(tokenParts[1]);
-
-      return user;
-    }
-  }
-
-  return null;
```

Comment:
As above, this is handled in the Authentication flow, and shouldn't be part of Authorization.

If we don't have a `req.user`, this is indicative of a developer error, and results in a `500 Server error`